tensorflow_lite_support/python/task/core/task_utils.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utils functions used in Task Python API."""
+
+from tensorflow_lite_support.cc.task.core.proto import base_options_pb2
+from tensorflow_lite_support.python.task.core import task_options
+from tensorflow_lite_support.python.task.core.proto import configuration_pb2
+
+_ProtoBaseOptions = base_options_pb2.BaseOptions
+
+
+def ConvertToProtoBaseOptions(
```

Comment:
As discussed in https://github.com/tensorflow/tflite-support/pull/816/files#r871930350, please remove this for now.